src/suggestion.rs

```diff
@@ -135,6 +158,131 @@ impl<'s> fmt::Display for Suggestion<'s> {
             0
         };
 
+        // For long lines, we will trim the literal displayed to fit in the terminal
+        // The misspelled word shall always be shown with as much info as possible
+        // Misspelled words that are too long shall also be ellipsized
+
+        // The paddings give some space for the ` {} ...` and extra indentation and formatting:
+        // ```
+        // 65 |  ...  here, on the second most pointless site ever! Well, wander ont  ...
+        //    |                                                                  ^^^
+        //    |                                                                                                                                                                                                ^^^
+        //    | - not, Ont, int, on, onto, cont, font, or one of 9 others
+        //    |
+        //    |   Possible spelling mistake found.
+        //    |
+        // ```
+        const PADDING_OFFSET: usize = 6;
```

Comment:
can you add reasoning about the constant values?

Reply:
done! I actually had a bug as I was not considering the dynamically variable for `indent`, which affects where the formatting happening. For this reason, I had variants when testing the same test strings in different lines of the file: x, xx, xxx for x digits.

After considering this, I had to remove some of those variables as they were not right and also const. I have added some comments about to help illustrate, can be kept and can be removed it as you think is good.